oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/extractor/ExtractorSpec.java

```diff
@@ -215,6 +239,48 @@ public void metrics(@DelegatesTo(SampleBuilder.class) final Closure<?> cl) {
         }
     }
 
+    @SuppressWarnings("unused")
+    public void slowSql(@DelegatesTo(SlowSqlSpec.class) final Closure<?> cl) {
+        if (BINDING.get().shouldAbort()) {
+            return;
+        }
+        LogData.Builder log = BINDING.get().log();
+        if (log.getLayer() == null
+                || log.getService() == null
+                || log.getTimestamp() < 1) {
+            LOGGER.warn("SlowSql extracts failed, maybe something is not configured.");
+            return;
+        }
+        DatabaseSlowStatementBuilder builder = new DatabaseSlowStatementBuilder(namingControl);
+        builder.setLayer(Layer.nameOf(log.getLayer()));
+        builder.setTimeBucket(log.getTimestamp());
```

Comment:
Hi, timeBucket and timestamp are different format, so you might need conversion here